app/controllers/application_controller.rb

```diff
@@ -94,6 +94,10 @@ def current_form
     @current_form ||= Form.find(params[:form_id])
   end
 
+  def current_live_form
```

Comment:
Should this have a test now that it's a public method?

Reply:
I don't think so, We don't test `current_form` and when it's called we stub out the request in the test it relates to so it's being tested indirectly. It's a case of testing behaviour rather than implementation.